### DISCUSSION TITLE
Just a newbie question - how to build the quasar.esm.prod.js

### DISCUSSION THREAD

**Discussion by cyfung1031** [quasarframework/quasar#10772] [General - Components / Directives / etc] [answered]
Can anyone give me some guildline or reference about building the quasar.esm.prod.js from "quasar/src"? 
I would like to test and check the modification of src files.
But using `app.use(Quasar, config)` always refer to the "quasar/dist/quasar.esm.prod.js"

**Comment by pdanpdan** [comment, marked as answer]
Clone repo, cd in ui folder, yarn, yarn build and you will have the built files in dist
